### PR TITLE
O3-2395: Add documentation for enabling traces in E2E tests

### PR DESCRIPTION
**Pull request by DilshanNaveen**

### CODE REVIEW

**Comment by DilshanNaveen** [comment]
@jayasanka-sack

File: pages/docs/frontend-modules/testing.mdx [hunk 1]
```diff
@@ -267,6 +267,71 @@ We recommend installing the official Playwright [VS Code plugin](https://playwri
...
+      },
+    },
+    // Other project configurations...
+  ],
+};
+```
+
+4. Save the changes to your `playwright.config.ts` file.
+
+#### Accessing Traces
```
Comment (denniskigen) [inline pages/docs/frontend-modules/testing.mdx:317]:
```suggestion
### Accessing Traces
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thank you for your PR, @DilshanNaveen. We appreciate your efforts. For this section, we're focusing on how to view traces rather than their configuration. It would be great if you could integrate this guide under this heading: https://o3-docs.openmrs.org/docs/frontend-modules/testing#view-test-reports-from-github-actions--bamboo. Could you kindly make this adjustment? Thanks again!

**Comment by DilshanNaveen** [comment]
Dear @jayasanka-sack, thank you for your review and your feedback, are you asking me to ditch the configuration part and keep the accessing traces part under View test reports from GitHub Actions / Bamboo section?

**Review by jayasanka-sack — APPROVED** [review]
Thanks @DilshanNaveen ! This is exactly what is needed.

**Pull request merged**